src/components/Search/Search.tsx

```diff
@@ -19,7 +24,25 @@ function reducer(state:object, update: {type:string, change:any}) {
 
 const Search: React.FC = () => {
 	const [state, dispatch] = useReducer(reducer, initialState)
-	
+
+	useEffect(() => {
```

Comment:
I like that this is set up for us.

Reply:
Yeah this is gonna make life so much easier this weekend.

Reply:
Yes it will!